app/models/order_detail.rb

```diff
@@ -465,6 +465,10 @@ def complete!
     change_status!(OrderStatus.complete_status)
   end
 
+  def force_complete!
```

Comment:
I considered `#to_problem!` though I thought this name was more direct. This is used to get ongoing reservations flagged as "problem" though it's done indirectly.